src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -399,7 +399,7 @@ public static class Importer
          public static readonly string CloseAllTabsButThis = "close all tabs but this";
          public static readonly string CloseAllTabsToTheRight = "close all tabs to the right";
          public static readonly string UseFilterForImport = "Use the filters selected not only for visualization but also for importing the data";
-         public static readonly string Title = "Importer";
+         public static readonly string Title = "Import Observed data";
```

Comment:
pls change to "Import Observed **D**ata"

Reply:
Done already ;)